### DISCUSSION TITLE
Sort by referenced column?

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#338] [Q&A]
Hi Francesco

a Question: We reference in one collection another one. And in fireCMS we show plain text of this "other" column and not the cryptic firestore ID.
Example:
Collection City
Collectoin Country

In the collection City we reference Country and in firemcms should the country name e.g. "Spain". 

the problem: When we sort the city list by country, then it takes the firestore ID e.g. 6T3EtTtfb8g687ioGjqI and not "Spain". And therefore sorting is not correct for the editor

can this be adjusted? I mean, can we configure this somehow that it sorts by plain text country name?

**Comment by fgatti675** [comment]
I am afraid when you sort by a column it will always use the id for sorting.
The only workaround I can think of is that you use custom ids in the countries collection, so you can use the id `spain` instead of something cryptic like `6T3EtTtfb8g687ioGjqI`. That way you would get the ordering you are looking for

**Reply by Amtech281** [reply]
thanks @fgatti675 for your feedback. Yes, this solution came also to my mind.
is this a firestore limitation or could this be enhanced in firecms?

**Reply by fgatti675** [reply]
I am afraid this is a Firestore limitation!
